docs/usage/configuration-options.md

```diff
@@ -594,11 +594,37 @@ Renovate supports two options:
 - `none`: No release filtering (all releases allowed)
 - `strict`: If the release's constraints match the package file constraints, then it's included
 
-We are working on adding more advanced filtering options.
+More advanced filtering options may come in future.
 
-Note: There must be a `constraints` object in your Renovate config for this to work.
+Note: There must be a `constraints` object in your Renovate config, or constraints detected from package files, for this to work.
```

Comment:
```suggestion
<!-- prettier-ignore -->
!!! note
    There must be a `constraints` object in your Renovate config, or constraints detected from package files, for this to work.

```

Choose your adventure:

1. Make a proper `note` admonition -> apply suggestion
2. Drop the `Note:` part of the sentence, and rely on the `There must be ...` part to explain the requirement. -> edit suggestion and then apply
3. Do nothing.